src/pvecontrol/utils.py

```diff
@@ -4,20 +4,24 @@
 from prettytable import PrettyTable
 from collections import OrderedDict
 from humanize import naturalsize
+from enum import Enum
 
 
 # Pretty output a table from a table of dicts
 # We assume all dicts have the same keys and are sorted by key
 def print_tableoutput(table, sortby=None, filter=[]):
   x = PrettyTable()
   x.align = 'l'
-  x.field_names = table[0].keys()
+  x.field_names = [*table[0].keys(), "sortby"]
   for line in table:
+    sort_data = line[sortby]
```

Comment:
```suggestion
    sort_data = line[sortby]
```

You should catch KeyError exception to handle if --sort-by key doesn't exists

Reply:
There is a dedicated task in clickup, I will do it later while adding checks on other commands too.